branching.md

```
@@ -0,0 +1 @@
+# This is the branching strategy we need for Git.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)